### PR TITLE
Add AS2 deprecation notices

### PR DESCRIPTION
**Pull request by rkoron007**

### CODE REVIEW

File: docs/source/config.json
Comment (glasser) [inline docs/source/config.json:6]: I think we should include the EOL date in the message.

Comment (rkoron007) [reply docs/source/config.json:6]: Good call out! I'll probably use the following text:
```
"Apollo Server 2 is officially deprecated with end-of-life scheduled for 22 October 2023. **Learn more about upgrading to a supported Apollo Server version.**
```

I'm working with DXE right now just to ensure we can see what it looks like in a deploy preview before we merge it :)

File: netlify.toml
Comment (glasser) [inline netlify.toml:1]: Just making sure this file addition was intentional

File: docs/source/testing/graphql-playground.mdx
Comment (glasser) [inline docs/source/testing/graphql-playground.mdx:10]: This is showing up as text. Maybe needs to be first or maybe needs to be unindented or both

**Review by glasser — APPROVED** [review]
lgtm if you fix the import thing

File: netlify.toml
Comment (rkoron007) [reply netlify.toml:1]: Yes, this will enable the `version-2` branch to build deploy previews going forward (we can also remove the file if you'd rather it wasn't in there)!

**Pull request merged**
